README.md

```diff
@@ -35,9 +35,33 @@ fetchBodyTypes()
 | `optionServiceUrl`           | URL to [Options Service](https://option-service.preprod.carforyou.ch/swagger-ui/index.html)                         |
 | `analyticsServiceUrl`        | URL to [Analytics Service](https://carforyou-analytics-service.preprod.carforyou.ch/swagger-ui/index.html)          |
 | `userNotificationServiceUrl` | URL to [User Notification Service](https://user-notification-service.preprod.carforyou.ch/swagger-ui/index.html)    |
-| `tokenRefreshServiceUrl`     | URL to Auth Service used to refresh access tokens                                                                   |
 | `debug`                      | Set to `true` to `console.log` requests and API responses.                                                          |
 
+### Authenticated Requests
+
+Authenticated requests need to be handled in a different way depending on weather the request is performed server- or client-side.
+Client-Side, we can use the api client as a singelton to store the token. Server-side the access token needs to be passed to the request in order
+to not expose user credentials to the global module scope and ensure the request is performed with the correct
+
+#### Implementing authenticated requests
+
+To implement a authenticated request, simply pass the `isAuthorizedRequest` option to the request helper (fetch/post/put/delete-Data) and the api client will add the `Authorization` header. For server side requests, don't forget to pass the options down to the helper function so the consumer can pass the `accessToken` when invoking the request.
```

Comment:
```suggestion
To authenticate a request, simply pass the `isAuthorizedRequest` option to the request helper (fetch/post/put/delete-Data) and the api client will add the `Authorization` header. For server side requests, don't forget to pass the options down to the helper function so the consumer can pass the `accessToken` when invoking the request.
```

Reply:
reading this, I'd love to get rid of all the "if this then that"'s (refers to the comment above). maybe we can simplify server/client side handling?

I'm also a bit confused by passing `isAuthorizedRequest`. wouldn't we just pass the token and that would then authenticate the request?